src/Game.cpp

```diff
@@ -585,7 +585,7 @@ void Game::Init(bool firstTime) {
 	TCOD_renderer_t renderer_type = static_cast<TCOD_renderer_t>(Config::GetCVar<int>("renderer"));
 	if (firstTime) TCODConsole::initRoot(screenWidth, screenHeight, "Goblin Camp", fullscreen, renderer_type);
 	TCODMouse::showCursor(true);
-	TCODConsole::setKeyboardRepeat(500, 10);
+//	TCODConsole::setKeyboardRepeat(500, 10);

```

Comment:
If you're going to do this, which I don't recommend, file a bug now to come back and fix it, or you'll forget.